web/spec/aggregator_test.rb

```
@@ -0,0 +1,85 @@
+require "minitest/autorun"
+require_relative "../lib/aggregator"
+
+class AggregatorTest < Minitest::Test
+  def test_ピクセル数がチャンネル数と等しいときは恒等集約になる
+    aggregator = Aggregator.new(channel_count: 4, pixel_count: 4)
+    [3, 7, 11, 15].each_with_index do |rssi, i|
+      aggregator.update(i, 76_000_000 + i * 100_000, rssi)
```

RuboCop (1.86.1)

[warning] Wrap expressions with varying precedence with parentheses to avoid ambiguity.

(Lint/AmbiguousOperatorPrecedence)